src/multi_vector_simulator/utils/data_parser.py

```diff
@@ -390,10 +392,11 @@ def convert_epa_params_to_mvs(epa_dict):
 
                 if EMISSION_FACTOR not in dict_asset[asset_label]:
                     dict_asset[asset_label][EMISSION_FACTOR] = {VALUE: 0}
+
             dict_values[asset_group] = dict_asset
         else:
-            logging.warning(
-                f"The parameters {MAP_MVS_EPA[asset_group]} are not present in the parameters to be parsed into mvs json format"
+            logging.INFO(
+                f"The assets parameters '{MAP_MVS_EPA[asset_group]}' is not present in the EPA parameters to be parsed into MVS json format"
```

Comment:
Does this only parse the asset group or also the asset name?

Reply:
This message is only there if the whole asset group is missing